platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -501,6 +509,16 @@ public StandardOutputWire<GossipEvent> getPcesReplayerEventOutput() {
         return pcesReplayerWiring.eventOutput();
     }
 
+    /**
+     * Get the input wire that the app communication component uses to accept its input.
+     *
+     * @return the input wire that the app communication component uses to accept its input
+     */
+    @NonNull
+    public InputWire<ReservedSignedState> getAppCommunicationComponentInput() {
```

Comment:
This getter shouldn't be necessary with the described changes, since all wiring will be done in this class

Reply:
updated.